api/v4/source/custom_profile_attributes.yaml

```diff
@@ -218,6 +218,35 @@
           $ref: "#/components/responses/Unauthorized"
         "403":
           $ref: "#/components/responses/Forbidden"
+  "/api/v4/custom_profile_attributes/group":
```

Comment:
more of a question of trying to understand, should we have an endpoint like this per feature?
This could be fine, but I'm wondering if we should have a more generic endpoint to which we pass a feature identifier for future use of plugins/features
(this one is still great ;) )

Reply:
This is one of those things that we'll probably generalize down the line, yes, but I want to see more features requiring this before taking that step :)